launcher-scripts/Linux-launcher.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 OLD_JAVA_HOME=$JAVA_HOME
-export JAVA_HOME=`pwd`/jdk8u212-b03-jre
-"$JAVA_HOME/bin/java" -DLAUNCHER_JAVA_PATH="$JAVA_HOME/bin/java" -jar ./libs/admin.jar
-export JAVA_HOME=$OLD_JAVA_HOME
+SCRIPT_DIRECTORY=`(cd $(dirname $0); pwd)`
```

Comment:
My first try was simply using `dirname $0` and this worked fine for the launcher, but it failed to open the Admin Console because `dirname $0` only returned a relative path. As a result, `-DLAUNCHER_JAVA_PATH ` can't reference to the right folder of the provided JRE.